bode/bode/models/task_relation.py

```diff
@@ -103,6 +103,12 @@ def get_rhs_related_tasks(task_id, filters=list()):
     def get_related_tasks(task_id):
         return TaskRelation.get_lhs_related_tasks(task_id) + TaskRelation.get_rhs_related_tasks(task_id)
 
+    def is_task_blocked(task_id):
+        for relation, task in TaskRelation.get_rhs_related_tasks(task_id):
```

Comment:
There is a cute bug in the application. Have a look:
![image](https://user-images.githubusercontent.com/72392522/167792219-de90667e-faf0-473e-aa53-73a2581c993b.png)
![image](https://user-images.githubusercontent.com/72392522/167792263-4c50483d-7e6a-430d-b50d-fa97263c924d.png)
![image](https://user-images.githubusercontent.com/72392522/167792290-7923b387-9adf-42c7-8c04-f085277af40c.png)
Fix is simple:
```suggestion
        for relation, task in TaskRelation.get_lhs_related_tasks(task_id):
```